### PR TITLE
Students UT - on test failures add indicative error

### PR DESCRIPTION
**Pull request by gal432**

### CODE REVIEW

File: lms/lmsdb/models.py [hunk 1]
```diff
@@ -411,6 +411,9 @@ def get_by_exercise(cls, exercise: Exercise):
 
 
 class ExerciseTestName(BaseModel):
+    FATAL_TEST_NAME = 'fatal_test_failure'
+    FATAL_TEST_PRETTY_TEST_NAME = 'כישלון פטאלי'
```
Comment (yammesicka) [inline lms/lmsdb/models.py:415]: "כישלון חמור"

File: lms/lmstests/public/unittests/services.py [hunk 2]
```diff
@@ -58,7 +65,20 @@ def _generate_python_code(self) -> str:
...
+        if junit_results:
+            results = junitparser.TestSuite.fromstring(junit_results)
+        if not results:
+            self._logger.info('junit invalid results (%s) on solution %s',
+                              junit_results, self._solution_id)
+            models.SolutionExerciseTestExecution.create_execution_result(
+                solution=self._solution,
+                test_name=models.ExerciseTestName.FATAL_TEST_NAME,
+                user_message='אנא פנה לסגל.',
+                staff_message='אחי, בדקת את הקוד שלך?',
```
Comment (yammesicka) [inline lms/lmstests/public/unittests/services.py:78]: 😂

File: lms/lmstests/public/unittests/services.py [hunk 3]
```diff
@@ -58,7 +65,20 @@ def _generate_python_code(self) -> str:
...
+        results = None
+        if junit_results:
+            results = junitparser.TestSuite.fromstring(junit_results)
+        if not results:
+            self._logger.info('junit invalid results (%s) on solution %s',
+                              junit_results, self._solution_id)
+            models.SolutionExerciseTestExecution.create_execution_result(
+                solution=self._solution,
+                test_name=models.ExerciseTestName.FATAL_TEST_NAME,
+                user_message='אנא פנה לסגל.',
```
Comment (yammesicka) [inline lms/lmstests/public/unittests/services.py:77]: "פנו", זו"נ

File: lms/tests/test_exercise_unit_tests.py [hunk 4]
```diff
@@ -14,21 +14,37 @@ def foo(bar=None):
 '''
 
 EXERCISE_TESTS = os.path.join(conftest.SAMPLES_DIR, 'student_test_code.py')
+INVALID_EXERCISE_TESTS = os.path.join(
```
Comment (yammesicka) [inline lms/tests/test_exercise_unit_tests.py:17]: I think pathlib can make more sense here, but ayw

Comment (gal432) [reply lms/tests/test_exercise_unit_tests.py:17]: But I only need the file path here, not the module itself

**Pull request merged**